tb_plugin/torch_tb_profiler/profiler/range_utils.py

```diff
@@ -1,3 +1,43 @@
+from enum import IntEnum
+
+
+EndpointTypes = IntEnum('EndpointTypes', ['START', 'END'], start=0)
+
+
+class EndPoint(object):
+    def __init__(self, ep_time, ep_pt_type, ep_value):
+        self.time = ep_time
+        self.pt_type = ep_pt_type
+        self.value = ep_value
+
+
+# src_ranges: item of ((start_time, end_time), value)
```

Comment:
could the src_ranges to be type tuple(start_time, end_time, value) to remove the embedded extra tuple? In this way, we can write:
for start_time, end_time, value in src_ranges:
    xxxx

Reply:
Cool! Done.